packages/cashier/src/Components/recent-transaction.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { epochToMoment } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+
+const RecentTransaction = ({ crypto_transactions, currency }) => {
+    const getStatus = status => {
+        switch (status) {
+            case 'CONFIRMED':
+                return 'Successful';
+            case 'LOCKED':
+                return 'In review';
+            case 'PERFORMING_BLOCKCHAIN_TXN':
+                return 'In process';
+            case 'PROCESSING':
+                return 'In process';
+            case 'SENT':
+                return 'Successful';
+            case 'VERIFIED':
+                return 'In process';
+            default:
+                return status[0].toUpperCase() + status.slice(1).toLowerCase();
+        }
+    };
+
+    let { address_hash, status_code, submit_date, transaction_hash, transaction_type } = crypto_transactions[
+        crypto_transactions.length - 1
+    ];
+
+    submit_date = epochToMoment(submit_date).format('MMM D, YYYY');
```

Comment:
Why is it `MMM`? 🤔 If it's month it should be `MM` right? And also `DD` so like `MM DD, YYYY`?

Reply:
MMM expresses the name of the month and MM its number and D works the same as DD